modules/installation-images-samples-disconnected-mirroring-assist.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-restricted-networks-preparations.adoc
+// * openshift_images/samples-operator-alt-registry.adoc
+// * openshift_images/configuring-samples-operator.adoc
+
+[id="installation-images-samples-disconnected-mirroring-assist_{context}"]
+= Cluster Samples Operator assistance for mirroring
+
+During installation, {product-title} creates a config map named `imagestreamtag-to-image` in the Cluster Samples Operator namespace. The `imagestreamtag-to-image` config map contains an entry, the populating image, for each image stream tag.
```

Comment:
s/Cluster Samples Operator namespace/openshift-cluster-samples-operator namespace
It's better to update to the namespace explicitly.

Reply:
Thank you! That makes sense.